### PR TITLE
Relaxation fix

### PR DESCRIPTION
**Pull request by KirbyBroderick**

### CODE REVIEW

File: configs/dask_cluster/dev_kube_cluster/worker-cpu.tmpl [hunk 1]
```diff
@@ -4,7 +4,7 @@ spec:
   containers:
   - image: ulissigroup/kubeflow:predictions
     imagePullPolicy: Always
-    args: [dask-worker, --nthreads, '1',--nprocs,'2', --no-dashboard, --memory-limit,'4 GiB',--death-timeout, '600']
+    args: [dask-worker, --nthreads, '1',--nprocs,'2', --no-dashboard, --memory-limit,'8 GiB',--death-timeout, '600']
```
Comment (zulissi) [inline configs/dask_cluster/dev_kube_cluster/worker-cpu.tmpl:7]: is it worth having a cpu relaxation worker? 4gb is plenty for other cpu workers if not doing inference

**Review by zulissi — COMMENTED** [review]
Misc small changes. I don't think this should be merged until there's also a relaxation test.

File: catlas/adslab_predictions.py [hunk 2]
```diff
@@ -60,9 +60,9 @@ def __init__(self, checkpoint, batch_size=8, cpu=False):
         config = torch.load(checkpoint, map_location=torch.device("cpu"))["config"]
 
         # Load the trainer based on the dataset used
-        if config["task"]["dataset"] == "trajectory_lmdb":
```
[the remark this answers is not in the document]
Comment (KirbyBroderick) [reply catlas/adslab_predictions.py:63]: This is for OCP's use, not ours:
https://github.com/Open-Catalyst-Project/ocp/blob/bc04a90aa01d14c16f77ef2d60290d2e2b0ee1d5/ocpmodels/common/relaxation/ase_utils.py#L114

File: configs/dask_cluster/dev_kube_cluster/worker-cpu.tmpl (hunk 1 above)
Comment (KirbyBroderick) [reply configs/dask_cluster/dev_kube_cluster/worker-cpu.tmpl:7]: https://github.com/ulissigroup/catlas/blob/main/configs/dask_cluster/dev_kube_cluster/worker-cpu.tmpl
Was the idea to request 9 and set the limit at 4 so workers would cut off if they started doing relaxations? If so we should probably make a new template for CPU relaxations

File: configs/dask_cluster/dev_kube_cluster/scheduler.yml [hunk 3]
```diff
@@ -1,14 +1,14 @@
...
-  affinity:
-    nodeAffinity:
-      requiredDuringSchedulingIgnoredDuringExecution:
-        nodeSelectorTerms:
-        - matchExpressions:
-          - key: network
-            operator: In
-            values:
-            - 10gb
+  # affinity:
```
Comment (zulissi) [inline configs/dask_cluster/dev_kube_cluster/scheduler.yml:3]: Maybe just delete these unless someone notices the scheduler really has to be on a fast network?

File: configs/dask_cluster/dev_kube_cluster/worker-cpu.yml [hunk 4]
```diff
@@ -0,0 +1,43 @@
+kind: Pod
```
Comment (zulissi) [inline configs/dask_cluster/dev_kube_cluster/worker-cpu.yml:1]: Unnecessary right? this comes from the template?

File: configs/dask_cluster/dev_kube_cluster/worker-cpu.tmpl [hunk 5]
```diff
@@ -4,7 +4,7 @@ spec:
   containers:
   - image: ulissigroup/kubeflow:predictions
     imagePullPolicy: Always
-    args: [dask-worker, --nthreads, '1',--nprocs,'2', --no-dashboard, --memory-limit,'4 GiB',--death-timeout, '600']
```
Comment (zulissi) [inline configs/dask_cluster/dev_kube_cluster/worker-cpu.tmpl:7]: Scheduling is easier if you just do one proc per pod; maybe cut everything in half?

File: configs/tests/test_cpu_relax.yml [hunk 6]
```diff
@@ -0,0 +1,30 @@
+dask:
+  partitions: -1 # if -1, 1 row per surface otherwise number of partitions
+
+memory_cache_location: '/home/jovyan/shared-scratch/catlas/test_cache_cpu_relax'
```
Comment (zulissi) [inline configs/tests/test_cpu_relax.yml:4]: do these really need their own caches?

**Review by zulissi — APPROVED** [review]
Looks pretty good! Added some comments but looks fine to merge after that.

File: configs/tests/test_cpu_relax.yml (hunk 6 above)
Comment (KirbyBroderick) [reply configs/tests/test_cpu_relax.yml:4]: Yeah, I want to wipe the cache and rerun things as needed, and I never want to keep any of these caches

**Pull request merged**